StructureDefinition/NHSDigital-MessageHeader.StructureDefinition.xml

```diff
@@ -1,9 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
-  <id value="35cdca0d-46a3-4dd3-9967-9470b864b92e" />
+  <id value="England-MessageHeader" />
   <url value="https://fhir.nhs.uk/StructureDefinition/England-MessageHeader" />
+  <version value="1.0.0" />
   <name value="EnglandMessageHeader" />
+  <title value="England MessageHeader" />
   <status value="draft" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="This profile defines the NHS England constraints and extensions on the UKCore FHIR resource [UKCore MessageHeader ](https://simplifier.net/hl7fhirukcorer4/ukcore-messageheader)" />
```

Comment:
Shouldn't cause a problem, but you have an extra space in the [ ] of the markdown link

UKCore FHIR, in text, should be referred to as FHIR UK Core